### PR TITLE
New Java options added

### LINKED ISSUE #155: Java tuning for Jetty
From the Shibboleht Wiki the following java options have been identified that should enhance the operation of IdPs. Some of there are in preparation for the introduction of the eduGAIN metadata feed and thus larger metadata files. Add these options to the start.ini file.

Bypass files validation for the SSL module, to work around a bug in Jetty 9.3.X

--skip-file-validation=ssl

Alternate garbage collector that reduces memory needed for larger metadata files

-XX:+UseG1GC

Maximum amount of memory that Jetty may use, at least 1.5G is recommended for handling larger (> 25M) metadata files but you will need to test on your particular metadata configuration. The AAF Production Metadata is currently about 2.5 Mb, this is future proofing for the eduGAIN metadata feed.

-Xmx1500m

### PR DESCRIPTION
**Pull request by trsau**
--skip-file-validation=ssl
-XX:+UseG1GC
-Xmx1500m

Tested on new install and upgrade.

resolves #155

### CODE REVIEW

File: templates/idp/start.ini [hunk 1]
```diff
@@ -24,3 +24,14 @@
...
+# Bypass file validation for the SSL module, to work around a bug in Jetty 9.3.X
+--skip-file-validation=ssl
+
+# Alternate garbage collector that reduces memory needed for larger metadata files
+-XX:+UseG1GC
+
+# Maximum amount of memory that Jetty may use, at least 1.5G is recommended
+# for handling larger (> 25M) metadata files but you will need to test on
+# your particular metadata configuration
+-Xmx1500m
```
Comment (rcaught) [inline templates/idp/start.ini]: My concern with this, is that you are setting a maximum amount of memory, when the problem at hand (as far as I can see) is that you want make sure enough memory is available.

After talking to @smangelsdorf, we may want to set both the minimum and maximum values like so: `-Xms1500m -Xmx1500m`.

This change could adversely affect VM's without enough allocated memory, so I suggest a reconsideration of the minimum memory requirement.  This should be noted / updated in the changelog and requirement files.  ie. http://ausaccessfed.github.io/shibboleth-idp-installer/requirements-checklist.html

Comment (trsau) [inline templates/idp/start.ini]: Initial heap size now set to equal max heap size.

**Comment by rcaught** [comment]
@trsau can you let me know how testing these changes went and I'll merge this in.

**Comment by trsau** [comment]
@rcaught These changes have been tested and are ready to be merged.

**Pull request merged**